samples/bluetooth/broadcast_config_tool/src/main.c

```diff
@@ -608,7 +707,10 @@ int main(void)
 	ERR_CHK_MSG(ret, "Failed to link zbus producers and observers");
 
 	ret = lc3_streamer_init();
-	ERR_CHK_MSG(ret, "Failed to initialize LC3 streamer");
+	if (ret) {
+		LOG_WRN("Failed to initialize LC3 streamer, transmitting fixed buffers");
```

Comment:
```suggestion
		LOG_WRN("Failed to initialize LC3 streamer. Transmitting fixed buffers.");
```
Watch out for those comma splices.